src/Features/Core/Portable/EditAndContinue/ManagedEditAndContinueCapabilities.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Microsoft.CodeAnalysis.EditAndContinue
+{
+    internal class ManagedEditAndContinueCapabilities
+    {
+        internal static readonly ManagedEditAndContinueCapabilities Net5CoreCLR = new(ManagedEditAndContinueCapability.Baseline, ManagedEditAndContinueCapability.AddDefinitionToExistingType, ManagedEditAndContinueCapability.NewTypeDefinition);
+        internal static readonly ManagedEditAndContinueCapabilities Net6CoreCLR = new(ManagedEditAndContinueCapability.Baseline, ManagedEditAndContinueCapability.AddDefinitionToExistingType, ManagedEditAndContinueCapability.NewTypeDefinition, ManagedEditAndContinueCapability.RuntimeEdits);
+        internal static readonly ManagedEditAndContinueCapabilities Net5MonoVM = new();
+        internal static readonly ManagedEditAndContinueCapabilities Net6MonoVM = new(ManagedEditAndContinueCapability.Baseline, ManagedEditAndContinueCapability.RuntimeEdits);
```

Comment:
We shouldn't need these in product code.

Reply:
Will move and remove some.. easier than getting naming right :)